.github/workflows/sync-cla.yml

```diff
@@ -0,0 +1,17 @@
+name: Sync CLA
+on:
+  push:
+    branches:
+      - 'main'
+jobs:
+  sync:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
+      - name: Sync gist
+        uses: pentatonicfunk/action-gist-repo-sync@7378b11353524fe9a357e6029a66b1e83203ed33 # v1.7
```

Comment:
Not sure if renovate/dependabot will pick this up, so maybe we can make a follow-up here to start using semver once the action has made a few releases and we see how it's being versioned? I assume you pinned to a hash as it doesn't have a real release cadence yet.

Reply:
Yes, Renovate supports this: https://docs.renovatebot.com/modules/manager/github-actions/#additional-information

I think we should pin it because it has access to a token.

Reply:
Ah, nice!